database/sqlite/sqlite_db_migration.c

```diff
@@ -79,6 +79,10 @@ const char *database_migrate_v5_v6[] = {
     NULL
 };
 
+const char *database_migrate_v7_v8_extra[] = {
+    "ALTER TABLE alert_hash ADD title text;",
```

Comment:
Are you sure that `title` can be NULL? :thinking: If it is not possible, please, add `NOT NULL` here.

Reply:
Hi Thiago! Thanks! Yes, it could be NULL. Although this current code will produce something for there, if a user goes back a version without title functionality and we have set this up as NOT NULL, then inserts might fail for his db.